warehouse/models/mart/gtfs/_mart_gtfs_fcts.yml

```diff
@@ -1030,6 +1034,22 @@ models:
         description: Part of a TripDescriptor.
       - name: trip_start_date
         description: Part of a TripDescriptor.
+      # foreign keys
+      - name: tu_gtfs_dataset_key
+        tests:
+          - relationships:
```

Comment:
Do we have a version of `dbt utils` that supports `relationships where`? could add a secondary check that it has the right data type but probably not really necessary

Reply:
Ah good call, will do